clean_hki_accidents.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+
+# File hki_accidents.csv contains all traffic accidents in Helsinki area from 2011 to 2016.
+# It has all the columns of the original dataset except Oslakpvm and Luovpvm.
+
+
+def drop_column(col):
+    if type(c) == int:
+        df.drop(df.columns[c], axis = 1, inplace = True)
+
+    else:
+        df.drop(c, axis = 1, inplace = True)
+
+
+
+df = pd.read_csv("data/hki_accidents.csv", encoding = 'utf-8')
+
+print("Original columns:\n")
+print(df.columns.values)
+
+# Clearly unnecessary columns
+
+# 0 = index, 33 = Saasel, 58 = Paallsel, 73 = Maankaytse, 76 = LisaKaisse, 90 = Paallksel
```

Comment:
Python 2.x doesn't support special characters in python files - not even in comments. Thus dropping these by referencing to their indexes.